README.md

```diff
@@ -0,0 +1,34 @@
+# Bootstrap Design Replication Project
+
+## Overview
+
+This project is a continuation of learning and strengthening my Front-end skills, improving code quality and better understanding of project workflows. Here I will replicate a chosen design from Boostrap as closely as possible using **HTML**, **SASS** and **Javascript** followed by **BEM** methodology.
+
+## Purpose
+I am building this project in order to deepen my understanding of:
+* **DRY**: Writing clean and reusable code
+* **Progressive Enhancement**: Building the project's solid, basic structure and progressively adding layers of styles and interactivity
+* **Mobile-First Apporach**: Designing layouts for smaller screens first and then enhancing them for larger resolutions
+* **Architecture**: I want to learn how to build and structure the project from scratch, without relying on tools like Webpack or Vite
+* **Rebase Flow**: In previous projects I used a merge-based workflow, so in this project I aim to learn and adopt rebase approach in order to improve my **Git** proficiency
+
+## Project Structure
+The project structure includes the following directories and files:
+```
+/dist           # Distibution folder for the final build
+/src            # Source files (.html, .scss, assets, etc.)
+README          # Project documentation
+build.sh        # Bash script for building the project into dist directory
+.gitignore      # Git ignore file
+```
+Although this project does not utilize Webpack, Vite or similar bundlers, I will write a custom **bash script** (build.sh) to transpile the application into the `/dist` folder. This approach will help me better understand project build proceesses and workflows.
```

Comment:
`to transpile the application` -> `to transpile styles and copy static assets`

Reply:
Alright, your suggestion seems clearer. I'll change that